### PR TITLE
[forge] add new capacity for cluster autoselection

### PR DESCRIPTION
**Pull request by rustielin**
### Description

This should balance 66% of the traffic to the new `forge-big-*` clusters, which can support 4x the max capacity. Once these are stable, we can fully roll out the big clusters and tear down the old ones .

Note: this change does not need to be made to the new forge python wrapper as that does cluster selection by regex from all running clusters, rather than selecting from a hardcoded list

### Test Plan

Spun up 3 new forge-big clusters, 2 of which will be used here. On each, I've run some load testing and can see that they can burst up to at least 600 nodes, barring intermittent txn emitter failures

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3432)

### CODE REVIEW

File: testsuite/run_forge.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ fi
 
 # available clusters to choose from
 # forge-1 is used for continuous testing exclusively
-FORGE_CLUSTERS=("aptos-forge-0" "aptos-forge-2")
+FORGE_CLUSTERS=("aptos-forge-big-0" "aptos-forge-big-1" "aptos-forge-0")
```
Comment (sitalkedia) [inline testsuite/run_forge.sh:30]: Is this used only for land blocking?

Comment (rustielin) [reply testsuite/run_forge.sh:30]: So here's the state of things:
* land-blocking tests will use the above cluster selection, from a list of hardcoded clusters
* continuous tests have a single hard-coded cluster. These are `aptos-forge-1` and `aptos-forge-2`: https://github.com/aptos-labs/aptos-core/blob/main/.github/workflows/continuous-e2e-tests.yaml
* pre-release tests are using the new python wrapper, and thus could schedule on any `aptos-forge-*` cluster: https://github.com/aptos-labs/aptos-core/blob/main/.github/workflows/pre-release-e2e-tests.yaml 
* the new shadow workflow does the same: https://github.com/aptos-labs/aptos-core/blob/main/.github/workflows/shadow-e2e-tests.yaml

Comment (rustielin) [reply testsuite/run_forge.sh:30]: So tl;dr after this PR lands, these new clusters are available for testing for land-blocking, shadow, and pre-release. @perryjrandall we should probably have a way to only have land-blocking tests on one cluster tho

**Review by sitalkedia — APPROVED** [review]

**Review by perryjrandall — APPROVED** [review]
fuckyeah

File: testsuite/run_forge.sh (hunk 1 above)
Comment (perryjrandall) [reply testsuite/run_forge.sh:30]: Yeah we just need a config somewhere, we could store in repo or store on the cluster somehow itself?

**Pull request merged**